### ISSUE TITLE
dockerTools.buildImage should accept the meta tag

### ISSUE THREAD

**Issue by kjeremy** [NixOS/nixpkgs#335877]
### Describe the bug
`dockerTools.buildImage` should accept the `meta` tag so that we could attach a `position` and `description` to docker derivations.

### Steps To Reproduce
1. ...
2. ...
3. ...

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.44, NixOS, 24.11 (Vicuna), 24.11.20240814.c3aa7b8`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.2`
 - channels(root): `"nixos"`
 - channels(jkolb): `""`
 - nixpkgs: `/nix/store/sfycwi72zfjsspidinx56ajaiffpyh17-source`

```
@yorickvP @roberth 
---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by yorickvP** [comment]
Good idea! In the meantime, you should be able to add it with `overrideAttrs`
